### PR TITLE
2022 lustre migration

### PR DESCRIPTION
**Pull request by jwlodek**

### CODE REVIEW

**Comment by jwlodek** [comment]
Update 2022-c2 PR should be merged first - this builds on that branch and changes all the paths for prosilica camera saving to lustre

File: azure-pipelines.yml [hunk 1]
```diff
@@ -3,10 +3,13 @@ resources:
...
 jobs:
-- template: azure-linux.yml@templates  # Template reference
-  parameters:
-    beamline_acronym: ARPES
+  - template: collection-2021-1.0.yml@templates
+    parameters:
+      beamline_acronym: ARPES
+  - template: collection-2021-1.2.yml@templates
+    parameters:
+      beamline_acronym: ARPES
```
Comment (mrakitin) [inline azure-pipelines.yml:15]: I think we can use the latest configs for these tests, such as in https://github.com/NSLS-II-TES/profile_simulated_hardware/blob/master/azure-pipelines.yml.

**Review by mrakitin — COMMENTED** [review]
I see no issues with the code changes but would like to have the CI tests passing (see below).

**Comment by mrakitin** [comment]
OK, let's do this: I'll update https://github.com/NSLS-II-ESM-1/profile_collection/pull/50, and then we rebase on `main` after the PR 50 is merged.

File: startup/41-ESM_motion.py [hunk 2]
```diff
@@ -636,14 +636,14 @@ def move_to(self, location):
 
 #The low temperature manipulator
 
-LT_manip=ESM_motion_device(os.environ['HOME']+
-    '/.ipython/profile_collection/startup/motion_definition_files/LT_manip_definition.csv',
-    'LOW TEMPERATURE MANIPULATOR')    
+LT_manip=ESM_motion_device(os.path.join(PROFILE_STARTUP_PATH,
```
Comment (maffettone) [inline startup/41-ESM_motion.py:639]: I would tend to encourage the use of `pathlib`, but I suppose this isn't an issue if we're not demanding interoperability between OS

**Review by maffettone — COMMENTED** [review]
All the code changes make sense. Minor gripe about pathlib, but it's not necessary.

File: startup/30-detectors.py [hunk 3]
```diff
@@ -90,7 +90,7 @@ def set_primary(self, n, value=None):
 class ESMbpm(ESMQuadEM):
     conf = Cpt(QuadEMPort, port_name="NSLS2_EM")
 
-
+#uncomment later and fix
```
Comment (mrakitin) [inline startup/30-detectors.py:93]: @maffettone, do we need this comment? Looks like everything works OK for QuadEM's.

File: startup/30-detectors.py [hunk 4]
```diff
@@ -100,11 +100,17 @@ class ESMbpm(ESMQuadEM):
...
+
+
 # qem09 not connected as of May 24, 2018
 # qem09 = ESMQuadEM('XF:21IDC-BI{EM:9}EM180:', name='qem09')
 ##qem10 = ESMQuadEM("XF:21IDC-BI{EM:10}EM180:", name="qem10")
 # qem11 not connected as of May 24, 2018
 # qem11 = ESMQuadEM('XF:21IDC-BI{EM:11}EM180:', name='qem11')
+
+
+#uncomment later and fix
```
Comment (mrakitin) [inline startup/30-detectors.py:113]: Same comment here.

**Review by mrakitin — APPROVED** [review]
Looks good to me besides the two odd comments (see below).

File: startup/30-detectors.py (hunk 3 above)
Comment (maffettone) [reply startup/30-detectors.py:93]: Don't think so. Not sure where it came from (potentially a bad copy/paste job) by the look of it.

**Comment by jwlodek** [comment]
Removed the two unneeded comments

**Review by mrakitin — APPROVED** [review]
Looks good to me now!

**Pull request merged**